packaging/setup/plugins/ovirt-engine-setup/ovirt-engine-grafana-dwh/core/config.py

```diff
@@ -220,7 +220,7 @@ def _customization_sso(self):
                 prompt=True,
                 default=True,
             )
-            with open(tmpconf, 'wb') as f:
+            with open(tmpconf, 'w') as f:
```

Comment:
Did you happen to find out what change caused this to be needed? Is it in paramiko? setup code? elsewhere?

Reply:
No idea